python/lsst/ts/Dome/task_scheduler.py

```diff
@@ -1,30 +1,36 @@
 import asyncio
 import logging
+import time
 
 run_status_loop = True
-logging.basicConfig(level=logging.INFO)
+logging.basicConfig(
+    format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=logging.INFO
+)
 log = logging.getLogger("TaskScheduler")
 
 
 async def schedule_task_periodically(period, task):
     """Schedules a task periodically with no drift.
 
+    I rewrote this example a bit to make it work for me:
+
+    https://stackoverflow.com/a/48204092
+
     Parameters
     ----------
     period: int
         The period in (decimal) seconds at which to schedule the function.
     task: coroutine
         The function to be scheduled periodically.
     """
-    loop = asyncio.get_event_loop()
     log.info(f"run_status_loop = {run_status_loop}")
 
     def g_tick():
-        t = loop.time()
+        t = time.time()
```

Comment:
Note that `time.monotonic` is preferred when you want to write something that measures elapsed time. (Credit to John Parejko for pointing that out to me.)

Reply:
I took out the bit of the code related to this comment but still thanks for the tip. I'll try not to forget it.